build/build-extensions.ps1

```diff
@@ -244,7 +244,15 @@ function CreateSiteExtensions() {
     New-Item -Itemtype directory -path $zipOutput -Force > $null
     if ($minorVersionPrefix -eq "10") {
         ZipContent $siteExtensionPath "$zipOutput\Functions.$extensionVersion$runtimeSuffix.zip"
-    } else {
+    } elseif ($minorVersionPrefix -eq "8") {
+        # The .NET 8 host doesn't require any workers. Doing this to save space.
+        Write-Host "Removing workers."
+        Remove-Item -Recurse -Force "$siteExtensionPath\$extensionVersion$runtimeSuffix\workers" -ErrorAction SilentlyContinue
```

Comment:
Do not want a hard failure here?

Reply:
i can change that -- but remember this is super-temporary anyway. we'd notice a huge difference in size if this didn't happen...